frontends/verilog/verilog_frontend.cc

```diff
@@ -100,6 +100,10 @@ struct VerilogFrontend : public Frontend {
 		log("    -assert-assumes\n");
 		log("        treat all assume() statements like assert() statements\n");
 		log("\n");
+		log("    -nodisplay\n");
```

Comment:
`-noprint` is a better name for the option.

Reply:
Depends on whether you think of `$print` cells or SystemVerilog "display tasks", I guess?
I can change it to `-noprint`, of course.

Reply:
There is not just `$display` but also `$write` and friends. What they do is print to the terminal, so I think `-noprint` accurately describes what the option does and is less ambiguous than `-nodisplay`.

Reply:
Just a thought, as there are no `$print` cells involved during elaboration, but only "display tasks", do you still want the option changed from `-nodisplay` to `-noprint`?

Reply:
Yes, because it's not about `$print` cells, but covering `$write` as well as `$display`.

Reply:
Sure, my point is that `$display*`, `$strobe*`, `$write*`, and `$monitor*` are all called "display tasks" in the LRM.

Reply:
Oh, I see your point now. I agree it doesn't need changing in this case.